src/config/ConfigManager.cpp

```diff
@@ -44,6 +44,7 @@ void CConfigManager::init() {
     m_config.addSpecialConfigValue(name, "shadow_boost", Hyprlang::FLOAT{1.2});
 
     m_config.addConfigValue("general:disable_loading_bar", Hyprlang::INT{0});
+    m_config.addConfigValue("general:disable_text_trim", Hyprlang::INT{0});
```

Comment:
can you flip the option? (text_trim and default 1)

Reply:
the idea was to be coherent with other general options that are 0 by default, are you sure I should flip it?

Reply:
yea, people complain about having both `xxx = bool` and `disable_yyy = bool` options and call it confusing. People generally prefer `enable_xxx` options it seems.

Reply:
done